src/runtime/dsn.layer2_types.cpp

```diff
@@ -13,1088 +13,1066 @@
 
 namespace dsn {
 
-int _kapp_statusValues[] = {app_status::AS_INVALID,
-                            app_status::AS_AVAILABLE,
-                            app_status::AS_CREATING,
-                            app_status::AS_CREATE_FAILED,
-                            app_status::AS_DROPPING,
-                            app_status::AS_DROP_FAILED,
-                            app_status::AS_DROPPED,
-                            app_status::AS_RECALLING};
-const char *_kapp_statusNames[] = {"AS_INVALID",
-                                   "AS_AVAILABLE",
-                                   "AS_CREATING",
-                                   "AS_CREATE_FAILED",
-                                   "AS_DROPPING",
-                                   "AS_DROP_FAILED",
-                                   "AS_DROPPED",
-                                   "AS_RECALLING"};
-const std::map<int, const char *> _app_status_VALUES_TO_NAMES(
-    ::apache::thrift::TEnumIterator(8, _kapp_statusValues, _kapp_statusNames),
-    ::apache::thrift::TEnumIterator(-1, NULL, NULL));
+int _kapp_statusValues[] = {
```

Comment:
I remember this `Autogenerated` diff has been not upload? @hycdong

Reply:
Actually, not all `thrift` files are generate `.h` and `.cpp` files during compile process. This file `dsn.layers.thrift` will auto-generate offline and upload `.h` and `.cpp` files.
By the way, this pull request doesn't update it, but it used to, those files are not formated.